sport/app/football/feed/Competitions.scala

```diff
@@ -145,6 +145,17 @@ object CompetitionsProvider {
       startDate = Some(LocalDate.of(2022, 7, 1)),
       finalMatchSVG = Some("womens_euros_2022_badge"),
     ),
+    Competition(
+      "870",
+      "/football/womens-world-cup",
+      "Women's World Cup 2023",
+      "Women's World Cup 2023",
+      "Internationals",
+      showInTeamsList = true,
+      tableDividers = List(2),
+      startDate = Some(LocalDate.of(2023, 7, 20)),
+      finalMatchSVG = Some("world_cup_2022_badge"),
```

Comment:
Same badge?

Reply:
Yeah was just waiting on the new one from Harry, now received :)